mmf/utils/logger.py

```diff
@@ -426,17 +425,12 @@ def __init__(
             )
 
         self._wandb = wandb
-
-        self._wandb_init = dict(entity=entity, config=config, project=project)
-
-        wandb_params = config.training.wandb
-        with omegaconf.open_dict(wandb_params):
-            wandb_params.pop("enabled")
-            wandb_params.pop("entity")
-            wandb_params.pop("project")
-
-        init_kwargs = OmegaConf.to_container(wandb_params, resolve=True)
-        self._wandb_init.update(**init_kwargs)
+        self._wandb_init = dict(entity=entity, project=project)
```

Comment:
Thanks for your prompt reply @ebsmothers and for making the change yourself. Is the minor change is in this line where `config=config` is removed from the dict?

Earlier I was passing the raw config dictionary to this `dict` so that it can be logged to W&B and one can reproduce an experiment from the logged configuration. 

Do you think it won't be of much use to the users? Happy to know your thoughts.

Reply:
Good catch. Actually the change I referred to was more minor, just a removal of .keys(). Config should still be included, so I added it back. Thanks for pointing out the omission

Reply:
Yeah, I did notice the removal of `.keys()`. Thanks for adding back the config. So excited about how it progressed.